src/Client.php

```diff
@@ -51,6 +47,22 @@ public function getEvents(array $query = null)
         return $collection;
     }
 
+  /**
+   * Get all events by an optional query presented as string.
+   * 
+   * @param string|NULL $queryString
+   *
+   * @return \Itk\EventDatabaseClient\Collection
+   */
+    public function getEventsFromString(string $queryString = null)
+    {
+      $res = $this->request('GET', $queryString);
```

Comment:
It seems that value here is actually a “path” (which may contain a query string). Consider renaming `$queryString` to `$path`.